recipes-graphics/wayland/wayland-protocols_1.13.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Collection of additional Wayland protocols"
```

Comment:
Please use .imx as suffixes for the version. This allows to spot it is specific to the imx support.